README.md

```diff
@@ -79,6 +79,10 @@ This manual twiddling of environment vars is not ideal but given that the altern
 
 Note that the Dockerfiles used to build these vendored `boringssl` builds may be found in the respective vendor directories, and can serve as a reference for the build environment needed to generate FIPS-compliant ztunnel builds.
 
+## Development
+
+Please refer to [this](./LOCAL.md).
```

Comment:
Actually, this reminds me that it would actually be good to extend `LOCAL.md` with some quick help about the `cargo test` options we have, and the relevant `Makefile` targets people should use for local dev.


- `make presubmit`
- `make test`
- relevant Make flags like `BUILD_IN_CONTAINER` and `TEST_MODE=root`

if we add that stuff to `LOCAL.md` linking to it makes more sense.

Reply:
@bleggett you totally got me!

I am looking around for a long time and find the LOCAL.md, so I want to add the entry first for all developers first.

The make flags are not well documented, it is good to be more clear in the document.

I will do it.